src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageAction.java

```diff
@@ -18,12 +18,9 @@
 
     @Override
     protected ActionResult execute() throws EntityDoesNotExistException {
-
+        // this variable is used as a flag for tests
```

Comment:
Best to mention the variable's name where possible in comments like this to avoid confusion in the future.